src/TikaOnDotnet.TextExtractor/TextExtractionResult.cs

```diff
@@ -29,11 +29,9 @@ public override string ToString()
 			var builder = new StringBuilder("Text:\n" + Text + "MetaData:\n");
 
 			foreach (var keypair in Metadata)
-			{
 				builder.AppendFormat("{0} - {1}\n", keypair.Key, keypair.Value);
-			}
 
 			return builder.ToString();
 		}
-	}
+    }
 }
```

Comment:
I also shuffled the code around a lot because there was a lot of overhead in the OCR stuff that that other person made. There was imho no need to make a separate class just to add OCR support.